src/sagemaker/estimator.py

```diff
@@ -74,6 +74,8 @@
 
 logger = logging.getLogger(__name__)
 
+DEFAULT_CPU_INSTANCE_TYPE = "ml.m5.xlarge"
+
```

Comment:
Need default/dummy instance type for serverless endpoint deployment as instance type will need to be used for creating models. Same thing for the `model.py` files as well

Reply:
Resolved this in my forked repo, will include that in the new PR